web/components/templates/requestsV2/costPill.tsx

```diff
@@ -0,0 +1,74 @@
+import { useState } from "react";
+import ThemedModal from "../../shared/themed/themedModal";
+import Link from "next/link";
+import { clsx } from "../../shared/clsx";
+import { ArrowTopRightOnSquareIcon } from "@heroicons/react/24/outline";
+
+const CostPill = () => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const handleOpenModal = (
+    event: React.MouseEvent<HTMLSpanElement, MouseEvent>
+  ) => {
+    event.stopPropagation();
+    setIsOpen(true);
+  };
+
+  return (
+    <>
+      <span
+        onClick={handleOpenModal}
+        className={clsx(
+          "bg-gray-50 text-gray-700 ring-gray-200 dark:bg-gray-900 dark:text-gray-300 dark:ring-gray-800",
```

Comment:
are we going with orange or gray? I'm okay with either - but what do you think?

Reply:
@ScottMktn orange looks good and noticeable, I like it